requirements.txt

```diff
@@ -3,4 +3,5 @@ element-calcium-imaging==0.1.0b0
 element-lab==0.1.0b0
 element-animal==0.1.0b0
 element-session==0.1.0b0
-ipykernel==6.0.1
\ No newline at end of file
+element-interface[scanreader,sbxreader,suite2p] @ git+https://github.com/datajoint/element-interface.git
```

Comment:
Let's move this requirement to the Dockerfile and README.  Otherwise this package won't be able to be published on pypi.